dom/events/Event-dispatch-click.html

```diff
@@ -209,6 +209,48 @@
   input.dispatchEvent(new MouseEvent('click'));
 }, `disconnected radio should be checked from dispatchEvent(new MouseEvent('click'))`);
 
+test(() => {
+  const input = document.createElement("input");
+  input.type = "checkbox";
+  input.disabled = true;
+  input.dispatchEvent(new MouseEvent("click"));
+  assert_true(input.checked);
+}, `disabled checkbox should be checked from dispatchEvent(new MouseEvent("click"))`);
+
+test(() => {
+  const input = document.createElement("input");
+  input.type = "radio";
+  input.disabled = true;
+  input.dispatchEvent(new MouseEvent("click"));
+  assert_true(input.checked);
+}, `disabled radio should be checked from dispatchEvent(new MouseEvent("click"))`);
+
+test(t => {
+  const input = document.createElement("input");
+  input.type = "checkbox";
+  input.disabled = true;
+  const label = document.createElement("label");
+  label.append(input);
+  dump.append(label);
+  label.click();
+  assert_false(input.checked);
+  label.dispatchEvent(new MouseEvent("click"));
+  assert_false(input.checked);
+}, "disabled checkbox should not be checked from label click");
```

Comment:
It'd be ideal to separate this into two tests (one for .click() and one for dispatchEvent). Alternately, it would be acceptable to just add assertion messages to the two different `assert_false`s, so that one can distinguish the failures.

Reply:
Per my latest thoughts at https://github.com/whatwg/html/pull/5805#issuecomment-672925220 I think now we should just remove these tests, or maybe move them to a separate -tentative.html file.

Reply:
I'll just remove them since Firefox currently fail those tests. https://bugzilla.mozilla.org/show_bug.cgi?id=1653797